grammar/src/toplevel.grm

```diff
@@ -24,7 +24,7 @@ fileAnnotations
   ;
 
 fileAnnotation
-  : "@" "file" ":" ("[" annotationEntry+ "]" | annotationEntry)
+  : "@" "file" ":" ("[" annotation+ "]" | annotation)
```

Comment:
I think it should be `unescapedAnnotation` instead of `annotation`.
Probably we should also remove `(":" "file")?` part from `annotationPrefix`.